website/ember-cli-build.js

```diff
@@ -7,47 +7,54 @@
 
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
-module.exports = function (defaults) {
-  let app = new EmberApp(defaults, {
-    // Add options here
-    sassOptions: {
-      precision: 4,
-      includePaths: [
-        '../node_modules/@hashicorp/design-system-tokens/dist/products/css',
-      ],
-    },
-    // we need to add this or Ember Sass compilation will mess up the generated CSS
-    minifyCSS: {
-      options: {
-        advanced: false,
-      },
-    },
-    // https://cli.emberjs.com/release/advanced-use/asset-compilation/#fingerprintingandcdnurls
-    fingerprint: {
-      // override defaults to also include json files which our markdown is compiled to. without this images don't render properly.
-      replaceExtensions: ['html', 'css', 'js', 'json'],
-    },
-    'ember-prism': {
-      components: [
-        'apacheconf',
-        'bash',
-        'css',
-        'handlebars',
-        'http',
-        'javascript',
-        'json',
-        'markup-templating',
-        'ruby',
-        'scss',
-      ],
-      theme: 'dracula',
-      plugins: ['line-numbers', 'normalize-whitespace'],
-    },
-    // https://github.com/shipshapecode/prember-sitemap-generator#usage
-    prember: {
-      baseRoot: 'https://helios.hashicorp.design/',
+const environment = EmberApp.env();
+const isVercelProduction = environment.VERCEL_ENV === 'production';
+
+const appConfig = {
+  // Add options here
+  sassOptions: {
+    precision: 4,
+    includePaths: [
+      '../node_modules/@hashicorp/design-system-tokens/dist/products/css',
+    ],
+  },
+  // we need to add this or Ember Sass compilation will mess up the generated CSS
+  minifyCSS: {
+    options: {
+      advanced: false,
     },
-  });
+  },
+  // https://cli.emberjs.com/release/advanced-use/asset-compilation/#fingerprintingandcdnurls
+  fingerprint: {
+    // override defaults to also include json files which our markdown is compiled to. without this images don't render properly.
+    replaceExtensions: ['html', 'css', 'js', 'json'],
+    // in the algolia index we want to store the fingerprinted illustrations' paths only in "production"
+    exclude: isVercelProduction ? [] : ['assets/illustrations/**'],
```

Comment:
follow up question from this morning, but is there any benefit to fingerprinting these in preview apps (but not production or dev) or would it be cleaner to disable fingerprinting across the board until/unless we figure out some way of making this work in algolia (which _may_ be possible)

Reply:
> may be possible

thinking out loud here that this could work if:
* Assets are fingerprinted in production
* The index is re-built based on those assets so that the fingerprinted URL in algolia matches what is in the deployment

For non-production environments we could disable fingerprinting so these assets work consistently across local dev and preview apps.